lib/gulp-tasks/patterns-import.js

```diff
@@ -0,0 +1,72 @@
+/**
+ *  @fileOverview
+ * Uses Gulpjs to import html patterns
+ *
+ *  @author       Scott Nath
+ *
+ *  @requires     NPM:gulp
+ *  @requires     NPM:lodash.merge
+ *  @requires     NPM:pattern-importer
+ */
+
+var gulp = require('gulp'),
+  merge = require('lodash.merge'),
+  utility = require('../utility');
+
+/**
+ * Function to get default options for an implementation of patternlab-import
+ * @returns {Object}  options  an object of default patternlab-import options
+ */
+var getDefaultOptions = function () {
```

Comment:
@scottnath @ericduran
cc: @conortm @beynya @gvorbeck @nikkiana @rebmullin @sergesemashko
I thought about this some and looked at other code examples from the Node community, and the definition of mutable functions and variables outside of the exports property is pretty common. And my argument that they are global in scope was just plain wrong, although it was based on a real and relevant experience of broken functionality.

I want other opinions on the matter, then. My own case of broken functionality regarded the fact that __dirname gave different results when outside the exports property than when inside. And this exact scenario is hard to replicate because objects were instantiated and modules required from different places.